client/src/App.js

```diff
@@ -1,15 +1,22 @@
-
-import React from 'react';
-import './App.css';
-import Login from './views/Login';
-import SnackBarProvider from './context/SnackBarProvider';
+import React from "react";
+import "./App.css";
+import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
+import Login from "./views/Login";
+import Signup from "./components/SignUp";
+import SnackBarProvider from "./context/SnackBarProvider";
+import DashBoard from "./components/Dashboard/Dashboard"
 
 function App() {
   return (
     <SnackBarProvider>
-      <div className="App">
-        <Login />
-      </div>
+      <Router>
+        <Switch>
+          <Route exact path="/" component={Login} />
```

Comment:
Here Login component has two routes '/' and '/login'
Instead, use this and put it last in switch
```suggestion
          <Redirect exact from="/" to="/login" />
```